Update getting_started & sce_file_contents with celltyping instructions (#193)
AlexsLemonade/scpca-docs: docs/sce_file_contents.md

| `singler_results` | If cell typing with `SingleR` was performed, the full result object returned by `SingleR` annotation. Only present for `processed` objects |
| `singler_reference` | If cell typing with `SingleR` was performed, the name of the [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html) reference dataset used for annotation. Only present for `processed` objects |
| `singler_reference_label` | If cell typing with `SingleR` was performed, the name of the label in the reference dataset used for annotation. Only present for `processed` objects |

> | `singler_reference_version`  | If cell typing with `SingleR` was performed with ontology labels, the [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html) package version used for the reference dataset |

Reviewer: Just to note that I'm going to file a PR today with adding this into the SCE object. I am adding both package and version so it will be `celldex_1-12-1` or whatever the version is. So we might update this once we make a decision regarding how we are storing this information.

Author: Ok, I'll circle back once that PR is set then (we can block this PR as needed, probably dont _need_ to for this situation)

Author: And we're back! In the end, we did the following. Names here still need to be updated, but the components are not changed.

https://github.com/AlexsLemonade/scpca-nf/blob/b4e8cc860ea7a40f730fd6807e12f0c89f87dfb1/bin/add_celltypes_to_sce.R#L140-L144

https://github.com/AlexsLemonade/scpca-nf/blob/b4e8cc860ea7a40f730fd6807e12f0c89f87dfb1/bin/add_celltypes_to_sce.R#L189-L193

| `cluster_nn`        | The nearest neighbor parameter value used for the graph-based clustering. Only present for `processed` objects |
| `celltype_methods` | If cell type annotation was performed, a vector of the methods used for annotation. May include `"submitter"`, `"singler"` and/or `"cellassign"`. If submitter cell-type annotations are available, this metadata item will be present in all objects. Otherwise, this item will only be in `processed` objects |
| `singler_results` | If cell typing with `SingleR` was performed, the full result object returned by `SingleR` annotation. Only present for `processed` objects |
| `singler_reference` | If cell typing with `SingleR` was performed, the name of reference dataset used for annotation. Only present for `processed` objects |
| `singler_reference_label` | If cell typing with `SingleR` was performed, the name of the label in the reference dataset used for annotation. Only present for `processed` objects |

> | `singler_ref_source`  | If cell typing with `SingleR` was performed, the source of the reference dataset (default is [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html)). Only present for `processed` objects |
> | `singler_ref_version`  | If cell typing with `SingleR` was performed, the version of the reference dataset source, with periods replaced as dashes (`-`). Only present for `processed` objects |

Reviewer: I think because we are referring to celldex and PanglaoDB in the getting started docs, and because we really are only supporting those two, you can explicitly say that the version is from celldex.
```suggestion
| `singler_reference_source`  | If cell typing with `SingleR` was performed, the source of the reference dataset (default is [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html)). Only present for `processed` objects |
| `singler_reference_version`  | If cell typing with `SingleR` was performed, the version of `celldex` used to create the reference dataset source, with periods replaced as dashes (`-`). Only present for `processed` objects |
```

| `cellassign_predictions` | If cell typing with `CellAssign` was performed, the full matrix of predictions across cells and cell types. Only present for `processed` objects |
| `cellassign_reference` | If cell typing with `CellAssign` was performed, the name of the organ/tissue grouping for which marker genes were obtained. Only present for `processed` objects |

> | `cellassign_ref_source`  | If cell typing with `CellAssign` was performed, the source of the reference dataset (default is `PanglaoDB`). Only present for `processed` objects |

Reviewer: ```suggestion
| `cellassign_reference_source`  | If cell typing with `CellAssign` was performed, the source of the reference dataset (default is `PanglaoDB`). Only present for `processed` objects |
```

> | `cellassign_ref_version`  | If cell typing with `CellAssign` was performed, the version of the reference dataset source (default is `PanglaoDB`, whose versioning scheme is a date in ISO8601 format). Only present for `processed` objects |

Reviewer: ```suggestion
| `cellassign_reference_version`  | If cell typing with `CellAssign` was performed, the version of the reference dataset source. For references obtained from `PanglaoDB`, the version scheme is a date in ISO8601 format. Only present for `processed` objects |
```
